packages/fxa-content-server/app/styles/modules/_marketing.scss

```diff
@@ -50,6 +44,7 @@
       }
 
       html[dir='rtl'] & {
+        padding-right: 10px;
```

Comment:
@chenba Hmm, why this change? I can't see any `padding-left` for ltr...

Reply:
This is so the `h2` would align with the list items below; the `li`s have a bottom and right padding of 10px.

Reply:
> 
> 
> This is so the `h2` would align with the list items below; the `li`s have a bottom and right padding of 10px.

@chenba Ah I see. Visually this looks fine, but this is not on par with LTR because the list should be aligned with `main-content`, right now there's a 10px padding before as you said.
The correct fix imo would be to remove this rule, and add this block to the `.firefox-service` rule:
```css
html[dir='rtl'] & {
  padding: 0 0 10px 10px;
}
```
Should I file a new bug? If so, should it be on BMO or here?

Reply:
Sure.  File it on Github.  Thanks!